topi/python/topi/x86/dense.py

```diff
@@ -191,7 +191,6 @@ def _schedule_dense_pack_template(cfg, s, C):
     z, y, x = s[packedB].op.axis
     s[packedB].reorder(z, x, y)
     s[packedB].parallel(z)
-    s[packedB].vectorize(y)
```

Comment:
How about using `try_vec`?  It could let AutoTVM to discard large configuration using timeout parameter. And we could keep `vectorize(y)` if it is possible.

Reply:
Thanks for the tips. After took a look, I found out that it's not necessary to vectorize here.